packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,732 @@
+    <Hds::Alert @type="inline" @color="success" @title="A simple title" @description="A simple description text" />
+    <Hds::Alert
+      @type="inline"
+      @color="success"
+      @title="An alert with a very long title and a long description text that should go on multiple lines"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque erat elit, lacinia at magna eget, porttitor lobortis nulla."
+    />
+    <Hds::Alert @type="inline" @color="success" @title="An alert with just a title and no description text." />
+    <Hds::Alert @type="inline" @color="success" @description="An alert with no title and just a description text" />
+    <Hds::Alert
+      @type="inline"
+      @color="success"
+      @title="An alert with rich text description (HTML)"
+      @description="The description text can support HTML tags embedded in the string value, like <strong>strong</strong> and <em>em</em> as well as <code>code</code> and <a href='#'>links</a>."
+    />
+    <Hds::Alert
+      @type="inline"
+      @color="success"
+      @title="An alert with extra/custom content"
+      @description="In special cases, you can pass extra content to the alert using the <code>Generic</code> contextual component."
+      as |A|
+    >
+      <A.Generic>
+        <DummyPlaceholder @text="some generic content" @height="50" @background="#eee" />
+      </A.Generic>
+    </Hds::Alert>
+  </div>
+
+  <h4 class="dummy-h4">Actions</h4>
+  <div class="dummy-alert-sample-grid dummy-alert-sample-grid--wide-content">
+    <Hds::Alert
+      @type="inline"
+      @color="warning"
+      @title="Action passed as yielded component"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+      as |A|
+    >
+      <A.Button @text="Action" @color="secondary" />
+    </Hds::Alert>
+    <Hds::Alert
+      @type="inline"
+      @color="warning"
+      @title="With multiple actions passed as yielded components"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+      as |A|
+    >
+      <A.Button @text="Secondary" @color="secondary" />
+      <A.Button @icon="plus" @text="Tertiary" @color="tertiary" />
+      <A.Link::Standalone @icon="plus" @text="Standalone" href="#" @color="secondary" />
+    </Hds::Alert>
+    <Hds::Alert
+      @type="inline"
+      @color="warning"
+      @title="With actions and custom content"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+      as |A|
+    >
+      <A.Button @text="Action" @color="secondary" />
+      <A.Link::Standalone @icon="plus" @text="Action" href="#" @color="secondary" />
+      <A.Generic>
+        <div class="dummy-alert-sample-custom-content-after-actions">This for example could be extra text, specific for
+          a special use case.</div>
+      </A.Generic>
+    </Hds::Alert>
+  </div>
+
+  <h4 class="dummy-h4">Dismiss</h4>
```

Comment:
@didoo can we change the `@color` for all these alerts for something different? maybe neutral?
We are recommending setting critical alerts as persistent in the usage guidelines.